packages/twenty-front/src/modules/object-record/record-board/record-board-column/hooks/useAddNewOpportunity.ts

```diff
@@ -1,75 +1,124 @@
 import { RecordBoardContext } from '@/object-record/record-board/contexts/RecordBoardContext';
 import { RecordBoardColumnContext } from '@/object-record/record-board/record-board-column/contexts/RecordBoardColumnContext';
```

Comment:
I don't understand why there is so much logic in this file? Couldn't you plug it back into AddNewCard? It feels like you are duplicating code between the two and creating things specific to Opportunity that might not be needed (e.g. do we really need to create `recordBoardNewOpportunityByColumnIdSelector` or could we use the existing
`recordBoardNewRecordByColumnIdSelector` instead? Maybe I'm missing something)

Reply:
Yup! I felt the same way too!
Ill fix this!